control_protocol.md

```diff
@@ -0,0 +1,296 @@
+
+:::{mermaid}
+sequenceDiagram
+    CA ->> N1: COORDINATOR|N1.CA|SIGNOUT
+    participant N1 as N1.COORDINATOR
+    N1 ->> CA: N1.CA|N1.COORDINATOR|ACKNOWLEDGE
+    Note right of N1: Removes "CA" with identity "IA"<br> from address book
+    Note left of CA: Shall not send any message anymore except SIGNIN
+:::
+
+
+#### Communication with other Components
+
+The following two examples show, how a message is transferred between two components `CA`, `CB` via one or two Coordinators.
+Coordinators shall send messages from their DEALER socket to other Coordinator's ROUTER socket.
+
+Coordinators shall hand on the message to the corresponding Coordinator or connected Component.
+
+
+:::{mermaid}
+sequenceDiagram
+    alt full ID
+        CA ->> N1: N1.CB|N1.CA| Give me property A.
+    else only Component ID
+        CA ->> N1: CB|N1.CA| Give me property A.
+    end
+    participant N1 as N1.COORDINATOR
+    N1 ->> CB: N1.CB|N1.CA| Give me property A.
+    Note left of CB: Reads property A
+    CB ->> N1: N1.CA|N1.CB| Property A has value 5.
+    N1 ->> CA: N1.CA|N1.CB| Property A has value 5.
+:::
+
+
+:::{mermaid}
+sequenceDiagram
+    CA ->> N1: N2.CB|N1.CA| Give me property A.
+    participant N1 as N1.COORDINATOR
+    Note over N1,N2: N1 DEALER socket sends to N2 ROUTER
+    participant N2 as N2.COORDINATOR
+    N1 ->> N2: N2.CB|N1.CA| Give me property A.
+    N2 ->> CB: N2.CB|N1.CA| Give me property A.
+    Note left of CB: Reads property A
+    CB ->> N2: N1.CA|N2.CB| Property A has value 5.
+    Note over N1,N2: N2 DEALER socket sends to N1 ROUTER
+    N2 ->> N1: N1.CA|N2.CB| Property A has value 5.
+    N1 ->> CA: N1.CA|N2.CB| Property A has value 5.
+:::
+
+Prerequisite of Communication between two Components are:
+- Both Components are connected to a Coordinator and {ref}`signed in<sign-in>`.
+- Both Components are either connected to the same Coordinator (example one), or their Coordinators are connected to each other (example two).
+
+
+The following flow chart shows the decision scheme and message modification in a Coordinator `Co1` of Node `N1`.
+Its full ID is `N1.Coordinator`.
+`n0`, `nR` are placeholders for sender and recipient Node ID.
+`recipient` is a placeholder for the recipient Component ID.
+`IA` is the connection identity of the Component `N1.CA` (if it is directly connected to `Co1`) or of its Coordinator `n0.COORDINATOR`.
+`IB` is the connection identities of `N1.Recipient`.
+Bold arrows indicate message flow, thin lines indicate decision flow.
+Thin, dotted lines indicate decision flow in case of errors.
+Placeholder values are written in lowercase, while actually known values are begin with an uppercase letter.
+
+:::{mermaid}
+flowchart TB
+    C1([N1.CA DEALER]) == "nr.recipient|n0.CA|Content<br>(==nr.recipient|N1.CA|Content)" ==> R0
+    C0([n0.COORDINATOR DEALER]) == "nr.recipient|n0.CA|Content" ==> R0
+    R0[receive] == "IA|nr.recipient|n0.CA|Content" ==> Cn0{n0 == N1?}
```

Comment:
Maybe this can be simplified to "we are receiving a message `nr.recipient|ns.CA|Content`? Ultimately it does not matter if it comes from a Component or a Coordinator, I think. 
Maybe use `Is` as the (generic) sender identity? Does that make sense?

Reply:
Done